iron-overlay-manager.html

```diff
@@ -132,6 +145,17 @@
         element = null;
       }
       this._lastFocusedNodes.push(element);
+
+      // Ensure always-on-top overlay stays on top.
+      if (onTop) {
+        minimumZ = this.currentOverlayZ();
+        this._overlays.push(onTop);
```

Comment:
I would add a comment that `onTop` has already been spliced out of `this._overlays`, because it happens 25 lines up and it's not obvious. Or even better, can you do the splicing here?